tests/functional/conftest.py

```diff
@@ -65,6 +73,21 @@ def docker_compose_file(test_dir):
     return test_dir / "fixtures" / "docker-compose.yml"
 
 
+@pytest.fixture(scope="session")
+def docker_compose_project_name():
+    """Set a consistent project name to enable optional reuse of containers."""
+    return "pytest"
```

Comment:
What do you think about making it a bit more specific to avoid clashes in case people have the same idea with other local projects using `pytest-docker`? Unlikely, I know.
```suggestion
    return "pytest-python-gitlab"
```

Reply:
Yup, makes sense.